### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by LAN0206** [nnzhan/Graph-WaveNet#17]
AttributeError: 'NoneType' object has no attribute 'seek'.You can only torch.load from a file that is seekable. Please pre-load the data into a buffer like io.BytesIO and try to load from it instead.

How to solve this problem!Please!!!!!For helping !!!!!!

**Comment by lixus7** [comment]
hello！did you solve this problem？i have met it too. If you find the way, could you tell me how to solve it? thank you ！！！

**Comment by LAN0206** [comment]
No,i have not solved it yet...




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "nnzhan/Graph-WaveNet"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年9月1日(星期二) 晚上9:33
收件人:&nbsp;"nnzhan/Graph-WaveNet"<Graph-WaveNet@noreply.github.com&gt;;
抄送:&nbsp;"蓝"<1050776794@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [nnzhan/Graph-WaveNet] Question (#17)





 
hello！did you solve this problem？i have met it too. If you find the way, could you tell me how to solve it? thank you ！！！

**Comment by lixus7** [comment]
I gave up the test.py and use the testing part in the train.py to reach my goal.

**Comment by lixus7** [comment]
i think we need to set the defults parameters of checkpoint in test.py

**Comment by zwqnju** [comment]
> i think we need to set the defults parameters of checkpoint in test.py

Yes, just add '--checkpoint ./garage/XXX.pth', and test.py can run correctly.

**Comment by LAN0206** [comment]
> i think we need to set the defults parameters of checkpoint in test.py

thanks.I will try it later!

**Comment by LAN0206** [comment]
> > i think we need to set the defults parameters of checkpoint in test.py
> 
> Yes, just add '--checkpoint ./garage/XXX.pth', and test.py can run correctly.

ok  i will try it !thanks

**Comment by qiyanq** [comment]
> > > i think we need to set the defults parameters of checkpoint in test.py
> > 
> > 
> > Yes, just add '--checkpoint ./garage/XXX.pth', and test.py can run correctly.
> 
> ok i will try it !thanks


> > i think we need to set the defults parameters of checkpoint in test.py
> 
> Yes, just add '--checkpoint ./garage/XXX.pth', and test.py can run correctly.



> > i think we need to set the defults parameters of checkpoint in test.py
> 
> Yes, just add '--checkpoint ./garage/XXX.pth', and test.py can run correctly.

I don't quite understand how to do it. Can you be more specific？thanks

**Comment by qiyanq** [comment]
> > > i think we need to set the defults parameters of checkpoint in test.py
> > 
> > 
> > Yes, just add '--checkpoint ./garage/XXX.pth', and test.py can run correctly.
> 
> ok i will try it !thanks

I don't quite understand how to do it. Can you be more specific?thanks
